bakery/pipeline.js

```diff
@@ -81,7 +81,7 @@ const bakeryJob = {
     { get: 'output-producer', trigger: true, version: 'every' },
     reportToOutputProducer(Status.ASSIGNED),
     { get: 'cnx-recipes' },
-    taskLookUpBook,
+    taskLookUpBook({ bucket_name: env.S3_BUCKET }),
```

Comment:
@tomjw64 - Should this be `bucketName` instead of `bucket_name`? I seem to get a `echo -n "https://undefined.s3.amazonaws.com` in my task definition currently otherwise.

Reply:
good catch and good question. I think you are right @rnathuji